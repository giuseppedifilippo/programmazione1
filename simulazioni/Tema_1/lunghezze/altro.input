Pippo Pluto
e
Paperino


qui quo qua
ciao, a tutti
questi
qua.

	arrivederci