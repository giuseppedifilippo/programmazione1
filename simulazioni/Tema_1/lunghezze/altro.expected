1 : e a 
3 : qui quo qua 
4 : qua. 
5 : Pippo Pluto ciao, tutti 
6 : questi 
8 : Paperino 
11 : arrivederci 
